app/views/external_users/claims/case_details/_retrial_detail_fields.html.haml

```diff
@@ -7,16 +7,39 @@
     &nbsp;
 
   .form-row
-    .form-col
+    .form-col.form-col-one-half
       = f.gov_uk_date_field(:retrial_started_at, legend_text: t('.retrial_started_at'), legend_class: 'govuk-legend', id: "retrial_started_at", error_messages: gov_uk_date_field_error_messages(@error_presenter, :retrial_started_at))
+
     .form-col
-      = f.anchored_label t('.retrial_estimated_length'), 'retrial_estimated_length'
-      = f.number_field :retrial_estimated_length, step: 1, min: 1, class: 'form-control'
-      = validation_error_message(@error_presenter, :retrial_estimated_length)
+      %fieldset
+        %legend.govuk-legend |
+          = t('.estimated_retrial_length')
+          .form-date
+            %p.form-hint |
+              = t('.estimated_retrial_length_hint')
+
+        .form-group.form-group-day
+          = f.anchored_label t('.days')
```

Comment:
We need to continue providing the anchor name here, otherwise the error from the header will not link properly to this field. Should be like this:
`= f.anchored_label t('.days'), 'retrial_estimated_length'`